src/java/org/apache/cassandra/cql3/statements/schema/CreateKeyspaceStatement.java

```diff
@@ -87,10 +83,6 @@ public Keyspaces apply(Keyspaces schema)
         keyspace.params.validate(keyspaceName);
         Keyspaces keyspaces = schema.withAddedOrUpdated(keyspace);
```

Comment:
Nit: now we can directly return `schema.withAddedOrUpdated(keyspace)` without putting it in the `keyspaces` var.